MvvmCross/Binding/Droid/Views/MvxFilteringAdapter.cs

```diff
@@ -117,6 +118,16 @@ public override void NotifyDataSetChanged()
 
         public MvxFilteringAdapter(Context context) : base(context)
```

Comment:
Can you call the base class here and put the `CommonInitialize` code in there? Use `MvxAndroidBindingContextHelpers.Current()` to prevent have to add the parameter.

`public MvxFilteringAdapter(Context context) : base(context, MvxAndroidBindingContextHelpers.Current())`

Reply:
That is precisely what I would like to avoid: the `Current` goes to the registration stack, and there is none at the moment of the adapter creation (I explicitly pass a binding context in my `MvxAdapter` descendant class, but `MvxFilteringAdapter` does not have the appropriate constructor.